src/app/content/state/countdown/countdown.effects.ts

```diff
@@ -0,0 +1,47 @@
+import { Injectable } from '@angular/core';
+import { Actions, ofType, createEffect } from '@ngrx/effects';
+import {interval, EMPTY, Subject, switchMap, of} from 'rxjs';
+import { map, withLatestFrom, takeUntil } from 'rxjs/operators';
+import { Store, select } from '@ngrx/store';
+import * as countdownActions from './countdown.actions';
+import { AppState } from './countdown.state';
+
+@Injectable()
+export class CountdownEffects {
+
+  private pauseSubject = new Subject<void>();
+
+  startCountdown$ = createEffect(() => this.actions$.pipe(
+    ofType(countdownActions.startCountdown),
+    withLatestFrom(this.store.pipe(select('countdown'))),
```

Comment:
i would use concatLatestFrom here instead withLatestFrom as it's the official ngrx function for selecting stuff inside an effect, see https://ngrx.io/api/effects/concatLatestFrom